js/lib/Loader.js

```diff
@@ -231,19 +229,23 @@ Loader.prototype.listAvailableFiles = function(sync, cb) {
     return this.manifest;
 };
 
-Loader.prototype.addPath = function (path) {
-    if (!path) return;
-    this.includePath.unshift(path);
-    this.multiPaths = true;
+Loader.prototype.addPath = function (paths) {
+    if (!paths) return;
+    paths = ilib.isArray(paths) ? paths : [paths];
+    this.addPaths = paths;
+    this.isMultiPaths = true;
```

Comment:
What if the client calls this:

```
loader.addPath("a/b");
loader.addPath("c/d/e/f");
```
Then, this would fail.

This method should be:

```
In the constructor:
this.addPaths = [];

In this method:
if (!paths) return;
var newpaths = ilib.isArray(paths) ? paths : [paths];
this.addPaths = this.addPaths.concat(newpaths);
```

The `this.isMultiPaths` does not seem to be used any more

Reply:
I re-updated it. and in terms of  `this.isMultiPaths` is used here. https://github.com/iLib-js/iLib/blob/updateLoader/js/lib/Utils.js#L475